docs/modules/ROOT/pages/how-to-guides/proc_prefetching-dependencies-to-support-hermetic-build.adoc

```diff
@@ -0,0 +1,195 @@
+:_content-type: PROCEDURE
+
+[id="prefetching-dependencies-to-support-hermetic-build_{context}"]
+= Prefetching the package manager dependencies for the Hermetic build
+
+{ProductName} uses Cachi2 to prefetch the package manager dependencies for the <<supported-languages>>. Cachi2 is a wrapper around the package managers that makes it easy to prefetch and cache dependencies. It also helps generate a manifest of all dependencies included in your builds, ensuring transparency and maintainability. 
+
+To ensure proper usage of Cachi2, follow these steps for your project’s builds process:
+
+* *Define dependencies* - Explicitly declare dependencies in a lock file generated by the package manager. Cachi2 only fetches dependencies that are declared in this file.
+
+* *Reproducible builds* - Cachi2 enforces exact version matching for dependencies, including transitive dependencies. This ensures controlled and reproducible build dependencies.
+
+* *Secure builds* - While package managers use lock files to resolve dependencies automatically, it is crucial to verify the checksums of all the dependencies to safeguard against supply chain attacks.
+
+[#supported-languages]
+.Supported languages
+
+[cols="1,1"]
+|===
+|Language
+|Package Manager
+
+|Go
+|`gomod`
+
+|Python
+|`pip`
+|===
+
+== Enabling prefetch builds for `gomod`
+
+.Prerequisites
+* You have an upgraded build pipeline.
+* You have dependencies outside your git repository.
+
+.Procedures
+For each component that you want to build hermetically:
+
+. Navigate to your component's source code.
+. Open the *.tekton* directory and locate the *.yaml* files related to the *pull-request* and *push* processes.
+. *Set up dependency prefetching with Cachi2:* In both the `.yaml` files, add the following prefetch-input pipeline parameter.
+
++
+[source,yaml]
+----
+param:
+    -  name: prefetch-input
+       value: <package-manager-lock-file> <1>
```

Comment:
We are specifically describing the behavior for gomod, so we don't have to generalize it to an arbitrary package manager.

Also, this description is contrasting to the `<1>` below as the value may not be a path to a lock file, instead, it may just be the value `gomod`.